src/clr-addons/mobile.scss

```diff
@@ -70,4 +70,32 @@
   .card {
     margin-top: 0.5rem;
   }
+
+  .signpost {
+    .signpost-content {
```

Comment:
tested it with signpost position right-middle and it changed the position of the signpost.
why is all this stuff even needed? visually it only changes the width, for which the 'width css' is enough